InvenTree/InvenTree/status_codes.py

```diff
@@ -269,38 +265,42 @@ class StockHistoryCode(StatusCode):
     RETURNED_FROM_CUSTOMER = 105
 
     options = {
-        LEGACY: _('Legacy stock tracking entry'),
-
-        CREATED: _('Stock item created'),
-
-        EDITED: _('Edited stock item'),
-        ASSIGNED_SERIAL: _('Assigned serial number'),
-
-        STOCK_COUNT: _('Stock counted'),
-        STOCK_ADD: _('Stock manually added'),
-        STOCK_REMOVE: _('Stock manually removed'),
-
-        STOCK_MOVE: _('Location changed'),
-
-        INSTALLED_INTO_ASSEMBLY: _('Installed into assembly'),
-        REMOVED_FROM_ASSEMBLY: _('Removed from assembly'),
-
-        INSTALLED_CHILD_ITEM: _('Installed component item'),
-        REMOVED_CHILD_ITEM: _('Removed component item'),
-
-        SPLIT_FROM_PARENT: _('Split from parent item'),
-        SPLIT_CHILD_ITEM: _('Split child item'),
-
-        MERGED_STOCK_ITEMS: _('Merged stock items'),
-
-        SENT_TO_CUSTOMER: _('Sent to customer'),
-        RETURNED_FROM_CUSTOMER: _('Returned from customer'),
+        LEGACY: _("Legacy stock tracking entry"),
+        CREATED: _("Stock item created"),
+        EDITED: _("Edited stock item"),
+        ASSIGNED_SERIAL: _("Assigned serial number"),
+        STOCK_COUNT: _("Stock counted"),
+        STOCK_ADD: _("Stock manually added"),
+        STOCK_REMOVE: _("Stock manually removed"),
+        STOCK_MOVE: _("Location changed"),
+        INSTALLED_INTO_ASSEMBLY: _("Installed into assembly"),
+        REMOVED_FROM_ASSEMBLY: _("Removed from assembly"),
+        INSTALLED_CHILD_ITEM: _("Installed component item"),
+        REMOVED_CHILD_ITEM: _("Removed component item"),
+        SPLIT_FROM_PARENT: _("Split from parent item"),
+        SPLIT_CHILD_ITEM: _("Split child item"),
+        MERGED_STOCK_ITEMS: _("Merged stock items"),
+        SENT_TO_CUSTOMER: _("Sent to customer"),
+        RETURNED_FROM_CUSTOMER: _("Returned from customer"),
+        BUILD_OUTPUT_CREATED: _("Build order output created"),
+        BUILD_OUTPUT_COMPLETED: _("Build order output completed"),
+        RECEIVED_AGAINST_PURCHASE_ORDER: _("Received against purchase order"),
+    }
```

Comment:
Could you place these changes into a separate style fix PR? Tracing why changes were made is difficult if there are multiple features / features + style fixes in one PR

Reply:
Fine)

Reply:
@matmair  I  have removed style guides from this request.